metricbeat/module/kubernetes/state_pod/_meta/testdata/kube-state-metrics-v1.3.0.plain

```diff
@@ -0,0 +1,464 @@
+# HELP go_gc_duration_seconds A summary of the GC invocation durations.
```

Comment:
Is this the same file as the one in `metricbeat/module/kubernetes/_meta/test/`?

Could we also add `metricbeat/module/kubernetes/_meta/test/kube-state-metrics` to this testdata? If we can I think we could remove `state_pod_test.go` as you suggested.